lib/taskScheduler.js

```diff
@@ -46,8 +46,8 @@ var TaskScheduler = function(config) {
       var capabilitiesSpecExcludes = ConfigParser.resolveFilePatterns(
           capabilities.exclude, true, config.configDir);
       capabilitiesSpecs = ConfigParser.resolveFilePatterns(
-          capabilitiesSpecs).filter(function(path) {
-              return capabilitiesSpecExcludes.indexOf(path) < 0;
+          capabilitiesSpecs).filter(function (path) {
+            return capabilitiesSpecExcludes.indexOf(path) < 0;
```

Comment:
I think leaving this the way it was before is better for now. If there's a stylistic issue here it is probably best handled in another PR.